### PR TITLE
Add js and py files for API pageswithprop

### PR DESCRIPTION
**Pull request by PRISCILAVILEMEN**
Created files that make requests to this API and printed the page titles as examples.

### CODE REVIEW

**Comment by PRISCILAVILEMEN** [comment]
Hi @brendajerop,
can you help me reviewing my first pr? :)

**Comment by PRISCILAVILEMEN** [comment]
> @PRISCILAVILEMEN Thank you for working on this!
> Some general comments:
> 
> * We need the PHP code as well, so please add it.
> * In your commit, make sure you add `API:Pageswithprop` to the readme files of Python, PHP and JavaScript folders.
> * Please provide a link to your sandbox page for this module.
> * We don't need to modify `.gitignore`.

@brendajerop , Thaks for the return..  

I want to know 
 1 * We need the PHP code as well, so please add it.
# I need to install something to run the code?

2- * In your commit, make sure you add `API:Pageswithprop` to the readme files of Python, PHP and JavaScript folders.
# about this task, I add the API, but the when click in the link, had error

3- * Please provide a link to your sandbox page for this module.
https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop

Is to Add this link?

**Comment by PRISCILAVILEMEN** [comment]
@brendajerop ,  I need to know where I input this link : https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop , Can I help me please ;)

**Comment by brendajerop** [comment]
> @brendajerop , I need to know where I input this link : https://www.mediawiki.org/wiki/User:Prisicilavilemen/Sandbox/API:Pageswithprop , Can you help me please ;)

In the pull request description or the comment section. You've already linked to it so that's fine :)

**Comment by PRISCILAVILEMEN** [comment]
@brendajerop hi,
I read this info at contributing.md (located on the root directory)
It says: 

"For GET requests modules _only_, python, javascript and php sample code can be auto-generated by following the instructions below:"

As this file says **can be auto-generated** not **should be auto-generated**, so should I update this file to avoid further misleading interpretations?

**Comment by PRISCILAVILEMEN** [comment]
I made the changes, please check, and give me feedback, thanks

File: php/get_pageswithprop.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+$url = $endPoint . "?" . http_build_query( $params );
+$ch = curl_init($url);
+
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+//curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
+
+$output = curl_exec( $ch );
+
+if (curl_errno($ch)) {
```
Comment (brendajerop) [inline php/get_pageswithprop.php]: What does this `if statement` do? Do we need it?

File: php/get_pageswithprop.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "list" => "pageswithprop",
+    "pwppropname" => "displaytitle",
+    "format" => "json"
+];
+
+$url = $endPoint . "?" . http_build_query( $params );
+$ch = curl_init($url);
+
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+//curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```
Comment (brendajerop) [inline php/get_pageswithprop.php]: Remove this comment

**Comment by PRISCILAVILEMEN** [comment]
@brendajerop , review my code ;) and give more feed backs

Best Regards

**Comment by brendajerop** [comment]
@PRISCILAVILEMEN Please rebase your changes on top of those in master. To do this, run `git pull --rebase upstream master`.

**Comment by brendajerop** [comment]
@PRISCILAVILEMEN What do you get when you run `git pull --rebase upstream master`?

**Comment by PRISCILAVILEMEN** [comment]
@brendajerop , Sorry my rebase was not complete, but now I updated my branch and merged, please check. thanks

**Comment by PRISCILAVILEMEN** [comment]
> @PRISCILAVILEMEN What do you get when you run `git pull --rebase upstream master?
My repo was a lot of commits behind, here is what I did before:
-  Rebased origin master from upstream master 
-  Rebased origin mw-pageswithprop from origin master

But it seems that all the changes weren't there.

I did rebase again on my branch directly from upstream master, and it seems that now it merged correctly.

This project is my first experience with git, but I think I am getting used to it.

Can you please give another look at the PR? 😄

**Comment by brendajerop** [comment]
@PRISCILAVILEMEN I've left some feedback on the discussion page of your sandbox: https://www.mediawiki.org/wiki/User_talk:Prisicilavilemen/Sandbox/API:Pageswithprop :)

**Pull request merged**